opencensus/trace_exemplar_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func writeTimeSeriesData(projectID string) error {
 	ctx := context.Background()
-	dataPoint := createDataPointWithExemplar()
+	dataPoint := createDataPointWithExemplar(projectID)
```

Comment:
The client used below should defer a client.Close()

Reply:
Fixed